### ISSUE TITLE
NS preview not working correctly

### ISSUE THREAD

**Issue by paulGeoghegan** [NativeScript/nativescript-cli#5790]
### Issue Description

I have been trying to use nativescript for quite a while now. I was trying to use it with svelte but gave up after encountering a seeming bug.
When I run "ns preview" everything seems fine but when I scan the QR code with my Android device I am presented with a blank screen. I then tried with just a js app but I have run in to the same problem.
It's honestly really frustrating because I really want to give Nativescript a go but I just can't get it to work.

### Reproduction

ns preview

### Environment

```yaml
OS: Windows 11 10.0.23606
CPU: (20) x64 Intel(R) Core(TM) i9-9900X CPU @ 3.50GHz
Shell: Not Found
node: 20.10.0
npm: 10.2.3
nativescript: Not Found
# android
java: Not Found
ndk: Not Found
apis: Not Found
build_tools: Not Found
system_images: Not Found
# ios
xcode: Not Found
cocoapods: Not Found
python: Not Found
python3: Not Found
ruby: Not Found
platforms: Not Found
```
### Dependencies
```json
"dependencies": {
  "@nativescript/core": "~8.6.0",
  "@nativescript/theme": "~3.0.2"
},
"devDependencies": {
  "@nativescript/preview-cli": "1.0.12",
  "@nativescript/webpack": "~5.0.18"
}
```

### Please accept these terms

- [X] I have searched the [existing issues](https://github.com/NativeScript/nativescript-cli/issues) as well as [StackOverflow](https://stackoverflow.com/questions/tagged/nativescript) and this has not been posted before
- [X] This is a bug report
- [X] I agree to follow this project's [Code of Conduct](https://github.com/NativeScript/NativeScript/blob/master/tools/notes/CONTRIBUTING.md#coc)

**Comment by GumnovRuslan** [comment]
Any update on this? I'm experiencing a very similar issue.

**Comment by rigor789** [comment]
Try with
```
"@nativescript/core": "~8.5.0"
```

There's a pending update to the Preview app that should work correctly on 8.6.0+.

**Comment by NathanWalker** [comment]
This is resolved in latest Preview updates for iOS and Android - newly released 8.7 and below is now supported.
